src/lib/datepicker/datepicker.md

```diff
@@ -51,6 +51,29 @@ year containing the `startAt` date.
 
 <!-- example(datepicker-start-view) -->
 
+#### Watching the views for changes on selected years and months
+
+When a year or a month is selected in `multi-year` and `year` views respecively, the `yearSelected`
+and `monthSelected` outputs emit the normalized chosen year and normalized month. By "normalized"
```

Comment:
...outputs emit a normalized date representing the chosen year or month. By "normalized" we mean that the dates representing a year will have their month set to January and their day set to the 1st. Dates representing months will have their day set to the 1st of the month.  For example, if...